### PR TITLE
README: add instructions for local hosting presentations

### PR DESCRIPTION
**Pull request by kortschak**
@btracey @elliott5 Please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ This is a repository for talks given by Gonum developers about Go and Gonum.
...
+## Running presentations locally
+
+Some slides in stacks may include playable code examples that depend on github.com/gonum tree imports, or other
+packages outside the standard library. To play these examples the presentations must be served locally using the
+present tool.
+
+The present tool is easily installed using go get:
+
+```
+go install golang.org/x/tools/cmd/present
```
Comment (sbinet) [inline README.md]: i'd spell `go get` as `go get` and not `go install` :)

ie:

``` sh
sh> go get -u golang.org/x/tools/cmd/present
```

(these are presumably instructions for beginners with no real grasp on the differences b/w `install` and `get`)

**Comment by sbinet** [comment]
LGTM (with minor comment)

File: README.md (hunk 1 above)
Comment (kortschak) [inline README.md]: Indeed.

**Comment by elliott5** [comment]
Well done @kortschak - that's better wording than I suggested.

**Pull request merged**